Dockerfile

```diff
@@ -162,13 +165,7 @@ RUN \
   touch /var/log/auth.log && \
   update-locale && \
   rm /etc/postsrsd.secret && \
-  rm /etc/cron.daily/00logwatch && \
-  if [[ ${VCS_VER} =~ ^refs/tags/v.+ ]]; then \
-    echo "export DMS_VERSION='${VCS_VER:11}'" >>/root/.bashrc; \
-  else \
-    echo "export DMS_VERSION='${VCS_REF}'" >>/root/.bashrc; \
-  fi
```

Comment:
Is this not useful to have available? I thought the main benefit of the `VERSION` file was for getting a version to compare against?

Being able to query the `DMS_VERSION` env var may still be useful vs reading a generic named `VERSION` file? I'm thinking in regards to other scenarios unrelated to this update check feature, or is that likely to be such a niche case that they should just `cat /VERSION`?

Reply:
> or is that likely to be such a niche case that they should just cat /VERSION?

IMO yes.

Now that we have the VERSION file, I thought this is not needed any more. If some really needs it in a var --> `foo=$(<VERSION)`